helper/kvdb/kvdb.go

```
@@ -0,0 +1,20 @@
+package kvdb
+
+type KVBatch interface {
+	Set(k, v []byte)
+	Write()
```

Must returns error when failed. We should not "swallow" any failure.

---

@DarianShawn 9d3272f

---

Got it. 🙌